spyder/plugins/editor/widgets/editor.py

```diff
@@ -2078,6 +2078,7 @@ def save_as(self, index=None):
         # The next line is necessary to avoid checking if the file exists
         # While running __check_file_status
         # See spyder-ide/spyder#3678 and spyder-ide/spyder#3026.
+        original_newly_created = finfo.newly_created

```

Comment:
Perhaps you can reformulate the comment above, because "next line" now refers to the new line you added.